frontend/src/components/form/TextInput.vue

```diff
@@ -41,6 +53,10 @@ input[type="password"] {
   transition-duration: 0.5s;
 }
 
+textarea.text-input {
+  height: 100px;
+}
```

Comment:
```suggestion
textarea.text-input {
  min-width: 100%;
  max-width: 100%;
  min-height: 100px;
}
```
resize시 부모 컴포넌트의 범위를 벗어나지 못하도록 수정하는 게 어떨까요?

Reply:
감사합니다 :)
앞으로도 css 리뷰 많이 부탁드려요!